vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/GeneratedVaadinComboBox.java

```diff
@@ -1445,7 +1444,6 @@ public GeneratedVaadinComboBox(T initialValue, T defaultValue,
             boolean acceptNullValues) {
         super("value", defaultValue, acceptNullValues);
         if (initialValue != null) {
-            setModelValue(initialValue, false);
```

Comment:
Why don't we need `setModelValue ` anymore? How otherwise the model value will be set?

Reply:
Is it initialised automatically in other place based on `value` property of the web component?

Reply:
The opposite question: why it should be used ?
And this is in fact a good question: why use extra method if everything works without it ?

There is no any failure in any IT/unit tests.
Which proves perfectly that this is extra method.

`setPresentationValue` updates both values: model is updated as a consequence of presentation value change.

Reply:
Yes, you're right, it updated in model as follows:
```
 setter.setElementValue(element, propertyName, presentationValue);
```
in `AbstractSinglePropertyField`. Then it's excessive.